### PR TITLE
[Merged by Bors] - fix: count long files, instead of printing all of them

### PR DESCRIPTION
**Pull request by adomani**
Reported on [Zulip](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Technical.20Debt.20Counters/near/466993718)

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by bryangingechen** [comment]
LGTM, but I'll let @grunweg hit the button on this.
bors d=[grunweg]

**Comment by bryangingechen** [comment]
Trying again...
bors d=grunweg

**Review by grunweg — COMMENTED** [review]
Sorry for causing this hiccup. I had not tested that change locally...
Fix looks good, broadly speaking. I have a refinement to suggest, and noticed that the comment directly above is also outdated.

File: scripts/technical-debt-metrics.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ done
 printf '%s|%s\n' "$(grep -c 'docBlame' scripts/nolints.json)" "documentation nolint entries"
 # the exceptions for files exceeding the 1500 line limit are split between the `longFile` and
 # the text-based linters.
-printf '%s|%s\n' "$(grep -c '^set_option linter.style.longFile [0-9]*' $(git ls-files '*.lean'))" "large files"
+printf '%s|%s\n' "$(grep '^set_option linter.style.longFile [0-9]*' $(git ls-files '*.lean') | wc -l)" "large files"
```
[the remark this answers is not in the document]
Comment (adomani) [reply scripts/technical-debt-metrics.sh]: I could not commit your suggestion: indeed GitHub allows you make it, but not to push it, for non-modified code.  :shrug:

**Comment by grunweg** [comment]
Thanks! Since I suggested the new code myself, let me
maintainer merge

**Pull request closed without merging**